### PR TITLE
fix(files): fixed aria-desciption logic for last breadcrumb

### LINKED ISSUE #41895: [BITV] 9.4.1.2/3.7 - In the breadcrumb navigation, there are "aria-label" attributes on "li" elements, which is not allowed, such as aria-label="Go to the "/" directory". (2)
https://report.bitvtest.de/default-en/cbedb5ef-4815-4cba-a83d-995599c178c3.html#checkpoint-bfbf07e5c3-v3-n7

9.4.1.2/11.7 - https://report.bitvtest.de/default-en/cbedb5ef-4815-4cba-a83d-995599c178c3.html#checkpoint-bfbf07e5c3-v11-n7

### PR DESCRIPTION
**Pull request by emoral435**
* Resolves: #41895 
## Summary
Cleaned up the aria-description logic a bit!

## Screenshots for the new changes
image

## Checklist

- Code is [properly formatted](https://docs.nextcloud.com/server/latest/developer_manual/digging_deeper/continuous_integration.html#linting)
- [Sign-off message](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md) is added to all commits
- [ ] Tests ([unit](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#unit-tests), [integration](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#integration-tests), api and/or acceptance) are included
- [x] Screenshots before/after for front-end changes
- [ ] Documentation ([manuals](https://github.com/nextcloud/documentation/) or wiki) has been updated or is not required
- [x] [Backports requested](https://github.com/nextcloud/backportbot/#usage) where applicable (ex: critical bugfixes)

### CODE REVIEW

File: apps/files/src/components/BreadCrumbs.vue [hunk 1]
```diff
@@ -31,10 +31,10 @@
 			dir="auto"
 			:to="section.to"
 			:title="titleForSection(index, section)"
-			:aria-description="ariaForSection(index, section)"
+			:aria-description="ariaForSection(section)"
 			@click.native="onClick(section.to)">
 			<template v-if="index === 0" #icon>
-				<Home :size="20"/>
+				<Home :size="20" :aria-label="t('files', 'Home')"/>
```
Comment (ShGKme) [inline apps/files/src/components/BreadCrumbs.vue:37]: The label should be on the interactive element itself — on the button. `NcBreadbrumb` sets it via the `name` prop. So the issue is probably in the data, where `name` "Home" is not translated here, in the `section` object.

**Review by ShGKme — CHANGES_REQUESTED** [review]

**Comment by emoral435** [comment]
Woops - pushed too soon. Will erase this comment once I find it, but the translation might be needed from another file etc. Will look into this!

File: apps/files/src/components/BreadCrumbs.vue (hunk 1 above)
Comment (emoral435) [reply apps/files/src/components/BreadCrumbs.vue]: That's odd. I checked, and when the `name` member in the `sections` object always will have the 'home' section translated:
image
So the fact that we even see the name 'Home' means that it should be translated, no?

**Comment by Pytal** [comment]
I just checked on master and it is already translated correctly 💡

image

**Comment by Pytal** [comment]
Seems @szaimen checked with German where it is not translated so no issue code-wise

**Comment by szaimen** [comment]
> Seems @szaimen checked with German where it is not translated so no issue code-wise

indeed, looks like it...

**Comment by emoral435** [comment]
@Pytal @ShGKme I still propose that this PR get merged. Upon checking the master branch, it appears that the last breadcrumb doesn't have that aria-description that we proposed to keep :/

image

This PR still fixes that! Updated the main summary for the PR though

**Review by Pytal — APPROVED** [review]

File: apps/files/src/components/BreadCrumbs.vue (hunk 1 above)
Comment (emoral435) [reply apps/files/src/components/BreadCrumbs.vue]: Turns out it was just due to the fact that the German translation is still untranslated, but the PR still holds value :)

**Comment by emoral435** [comment]
/skjnldsv-backport ea765cfdd35d7f935b82b9d4dd438015a1d9e7e0 to stable28

**Review by skjnldsv — APPROVED** [review]

**Pull request merged**